config/schema.yaml

```diff
@@ -0,0 +1 @@
+fields:
```

Comment:
Let's add a commented-out example or two?

Reply:
I'll do that within the docs folder. We'll be evolving it to ZK most likely so I am leaning to not adding it to the main repo yet.